modules/core/02-client/legacy/v100/solomachine.go

```diff
@@ -88,6 +88,13 @@ func (cs ClientState) ExportMetadata(_ sdk.KVStore) []exported.GenesisMetadata {
 	panic("legacy solo machine is deprecated!")
 }
 
+// VerifyClientMessage returns nil
+func (cs *ClientState) VerifyClientMessage(
```

Comment:
I wasn't too sure what we wanted to do here (if anything). 

What is the history behind this legacy solo machine anyway? :)